src/api/app/controllers/webui/projects/meta_controller.rb

```diff
@@ -12,8 +12,7 @@ def show
 
       def update
         authorize @project, :update?
-        updater = ::MetaControllerService::ProjectUpdater.new(project: @project, request_data: @request_data)
-        updater.call
+        updater = ::MetaControllerService::ProjectUpdater.new(project: @project, request_data: @request_data).call
 
         status = if updater.valid?
```

Comment:
why not using `@errors` in here? You can remove the updater variable and the changes to return self :nerd_face:

Reply:
squashed @Ana06

Reply:
@vpereira 

> I couldn't visualize what you are proposing.. could you please show me in code? :)

I can add a commit to this PR with the suggestion. Is that fine for you?

Reply:
There are several things here:

-  I don't understand what you gain making that `call` return  `self`. It has to be more expensive than returning `nil` and I don't see you get any improvement from it. Is it for the tests? If that is the case, I don't think you should change the code (change the tests to fit the code instead).
- You could return true or false and then you don't need to call `valid?`. Alternatively you could stop setting `@errors` in `call` (is not used) and leave the rest of the code as it was. [This refactoring would bring a small improvement, but if it only about the tests not related]
- `valid?` method is weird, why is it checking `@errors.blank?` [maybe also not the focus of the PR]

Reply:
@Ana06 thank you for your inputs. It's one way to design it. There is another way to have ```call``` as class method, which in turn, it calls ```new``` and return self anyway. 

For instance:

```
class CalculateVendorPayoutService
  def new(params)
    ...
  end

  def self.call(params)
    new(params)
  end
end
```

So because we used already this pattern in another place https://github.com/openSUSE/open-build-service/blob/master/src/api/app/services/project_configuration_service/project_configuration_updater.rb#L9  I think thats ok to keep like that.